docs/newuser.md

```diff
@@ -1,29 +1,41 @@
+
+## Platform Hub
+
+## Externally Accessible
 These are all services that you can access through normal internet access with personal accounts.
 ### GitHub
-[Github](https://github.com/UKHomeOffice) to access our repositories you must have your personal Github invited to the UK Home Office organisation on Github, someone with access can request this for you.
+[Github](https://github.com/UKHomeOffice) to access our repositories you must have your personal Github invited to the UK Home Office organisation on Github, someone with access can request this for you. Please note that you will have to have your full name as your name on github along with 2 Factor Authentication. If you don't have a personal account on Github then you will need to create one.
+
 ### Quay
-[Quay](https://quay.io) You can access this with your github account after requesting access to our Quay Organisation with your personal account. Requests should be raised on the [bau board](https://github.com/UKHomeOffice/application-container-platform-bau/issues)
+[Quay](https://quay.io) You can access this with your github account after requesting access to our Quay Organisation with your personal account. Requests can be made on the [Platform Hub](https://hub.acp.homeoffice.gov.uk/help/support/requests/overview)
 
-## Office 365 Account
-[Office 365](https://portal.office.com), these are provided by emailing DigitalCapabilitiesPMO@homeoffice.gsi.gov.uk, this is not managed by us.
+## Internally Accessible
 
 ### Artifactory
-[Artifactory](https://artifactory.digital.homeoffice.gov.uk.). Sign in with office 365 button. The docker repository in artifactory is accessed by the docker.digital.homeoffice.gov.uk address.
+[Artifactory](https://artifactory.digital.homeoffice.gov.uk.). Requests can be made on the [Platform Hub](https://hub.acp.homeoffice.gov.uk/help/support/requests/overview) Sign in with Office 365 button. The docker repository in artifactory is accessed by the docker.digital.homeoffice.gov.uk address.
+
 ### Gitlab
-[GitLab](https://gitlab.digital.homeoffice.gov.uk) To get access to this you will need to first login via office 365 by clicking the office 365 login button and and then request for your account to be unblocked.
-## VPN
-The VPN is used to be able to get through to our multiple AWS environments as well as other data centers. The profiles can be downloaded from [authd](https://authd.digital.homeoffice.gov.uk) which you can access using your office 365 SSO. The following services can be only accessed when connected to the Dev VPN profile.
+[GitLab](https://gitlab.digital.homeoffice.gov.uk) To get access to this you will need to first login via office 365 by clicking the office 365 login button requests can be made on the [Platform Hub](https://hub.acp.homeoffice.gov.uk/help/support/requests/overview)
```

Comment:
Need some sort of punctuation for between `office 365 login button` and `requests can be made` (unless I'm misreading it).

Also, it's not clear what kind of support request can/should be made to gain access? (Same point here about using a direct link to a support request form, to make it a tad more useful)